platform/podjs/README.md

```diff
@@ -20,21 +20,130 @@ feature. While we could support the creation of a container and a
+          "@polypoly-eu/rollup-plugin-copy-watch": "file:../../dev-utils/rollup-plugin-copy-watch",
+          ...
+       }
+    ```
+
+    With the above, we make sure the feature uses our custom [rollup-plugin-copy-watch](https://github.com/polypoly-eu/polyPod/blob/main/dev-utils/rollup-plugin-copy-watch/) plugin to watch the copied files. To be able to do so, add a `watch` script on your `package.json`:
+
+    ```json
+      "scripts": {
+         "watch": "rollup --watch -c",
+      },
+    ```
+
+    and run
+
+         $ npm run watch
+
+    when you desire to start watching the process.
+
+3.  Next step, you need to define your `rollup.config.js` at the same level as your `package.json`.
+
+    There you need to have the `copy` (of our `"@polypoly-eu/rollup-plugin-copy-watch"`) under the `plugins` section with the `targets`
+    setup for `src`, which in this case it's the `"node_modules/@polypoly-eu/podjs/dist/pod.js"`, and `dest` where it will be ended to (e.g. `dist`).
+    So, it looks like:
+
+```js
+   import copy from "@polypoly-eu/rollup-plugin-copy-watch";
+
+   export default {
+      input: "src/your-input-file.js",    // point to your source file
+      output: {
+         file: "dist/myfile.js", // point to your output file
+         format: "iife",     // the format of your output file, for now only iife is supported
+         globals: {
+               // where you define all the external packages you want to use
+               pod: "pod",
+            },
+         }
+      },
+      plugins: [
+         ...
+         copy({
+            targets: [
+            {
+               src: [
+                  "node_modules/@polypoly-eu/podjs/dist/pod.js",
+                  ...
+               ],
+               dest: "dist",
+            },
+         ]
+         }),
+      ]
+   }
+```
+
+This way you have copied successfully the `pod.js` file from the `@polypoly-eu/podjs` module, and you can use it in your feature.
+
+4.  So, to compile the input files and run the previous configurations,
+    you should define a `build` script in the `package.json` file,
+    which will run the `rollup` command. For example, for Typescript directories, you can use:
+
+```json
+   {
+      ...
+      "scripts": {
+         "build": "shx rm -rf dist && tsc && rollup -c rollup.config.js",
+         ...
+      },
+   }
+```
+
+5.  You are ready to build your feature. Try:
 
-## Usage
+    `$ npm run build`
 
-You will need to use this module for creating a polyPod-compatible
-feature. Follow these instructions:
+    and you should have a `dist` folder with your feature included the `pod.js` file.
 
-1. Add _podjs_ as a development dependency to your feature.
-2. Copy or link `node_modules/@polypoly-eu/podjs/dist/pod.js`.
-3. Include `pod.js` from your feature's main document - before the
-   feature's own code.
+6.  Don't forget to include the `pod.js` from your feature's main document - before the feature's own code.
```

Comment:
I think after the whole copying-the-`pod.js`-file part (which isn't that interesting how people do it) comes the interesting part, something like: 

1. Including it in the feature's HTML file(s) as a `script` - which features have to do any way, not just for using pod.js, that's the same for all polyPod implementations.
2. Opening the feature's HTML file in a regular browser, pointing out that `file:` URLs are supported, no need to run an HTTP server.
3. Pointing out that the feature's manifest needs to be exposed as `window.manifestData` in order to see the correct navigation bar colour and the correct (localised) feature name.
4. Pointing out that navigating back within a feature works through the browser's native back button.

Reply:
Thanks for this! I felt I was missing important parts but wasn't aware of this "testing" part as well :>
I mainly "rephrased" your comment into the Readme... let me know, for further rephrasing or missing parts :)